plugin/trino-exchange-filesystem/pom.xml

```diff
@@ -34,6 +34,10 @@
                     <groupId>com.nimbusds</groupId>
                     <artifactId>oauth2-oidc-sdk</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Enforcing newer version of `netty-tcnative-boringssl-static` is more future proof than exclusion. 
But I am not sure if we really use this dependency with our uses of `azure-identity`

Reply:
Sure. Since there is no bom I need to add dependency definition for all system/architecture pairs